ui/apps/platform/src/services/VulnerabilitiesService.js

```diff
@@ -5,7 +5,18 @@ import queryService from 'utils/queryService';
 import entityTypes from 'constants/entityTypes';
 import axios from './instance';
 
-const csvUrl = '/api/vm/export/csv';
+function getCSVExportUrl(cveType) {
+    if (cveType === entityTypes.CLUSTER_CVE) {
+        return '/api/export/csv/cluster/cve';
+    }
+    if (cveType === entityTypes.NODE_CVE) {
+        return '/api/export/csv/node/cve';
+    }
+    if (cveType === entityTypes.IMAGE_CVE) {
+        return '/api/export/csv/image/cve';
+    }
+    return '/api/vm/export/csv';
```

Comment:
Unlike `entityTypes.CVE` above which find in files can identify in a possible future clean up, the `return` statement is invisible. If the default endpoint will become either incorrect or obsolete, or both, what do you think about a comment to remember to clean up later?

This is a small enough file that rewrite in TypeScript would allow something like:

```ts
function getCSVExportUrl(cveType: 'CLUSTER_CVE' | 'IMAGE_CVE' | 'NODE_CVE') {
```

Reply:
Not in this pull request, but anything we can do ahead of time to strengthen reusable code reduces the chance time pressure squeezes it out when we write new vulnerability management pages in TypeScript.

Reply:
Yea that's worth a comment. Let me add that as well